src/graphql/syncCodePackages.ts

```diff
@@ -159,6 +160,17 @@ export async function syncCodePackages(
       ),
       concurrency,
     ),
+    // make sure all Repositories exist
+    syncRepositories(
```

Comment:
ah sorry - i thought i fixed it such that the repository would upsert to avoid the extra api calls here